src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -125,7 +125,7 @@ public function configure(array $configuration): void
         // optional <desc>
         $desc = '(?:\s+(?P<desc>\V*))';
 
-        $this->regex = '/'.$indentRegex.'\*\h*@(?J)(?:'.implode('|', $types).')'.$desc.'\h*\r?$/x';
+        $this->regex = '/'.$indentRegex.'\*\h*@(?J)(?:'.implode('|', $types).')'.$desc.'\h*\r?$/';
```

Comment:
TODO replace `\ ` to ` ` in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7050/files#diff-169db8086e78c086839aa0c02cc2513cbb0ef60f245406e40debe18f2acea0cdR128 once that PR is merged